setup.py

```diff
@@ -1,6 +1,33 @@
 #!/usr/bin/env python
 """
-  TODO: this
+<Program Name>
+  setup.py
+
+<Author>
+  Santiago Torres <santiago@nyu.edu>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  May 23, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  setup.py script to install in-toto framework and in-toto scripts
+
+  # System Dependencies
+    - Python2.7 (www.python.org)
+    - OpenSSL (www.openssl.org)
+    - git (git-scm.com)
+    - pip (pip.pypa.io)
+    - virtualenvs - optional but strongly recommended!
+      (http://docs.python-guide.org/en/latest/dev/virtualenvs/)
+
+  # Installation from GitHub
+    ```
+    pip install git+git://github.com/in-toto/in-toto@develop
```

Comment:
interesting that `git+git` exists as a protocol. I think I'd rather opt for `git+https`.

Reply:
Will change

Reply:
Just a note, I wasn't being pedantic, apparently you can do git+git :P

Reply:
I think https is a good choice. 
Btw. here's a good [overview of `git`'s protocols](https://git-scm.com/book/id/v2/Git-on-the-Server-The-Protocols) which are [all supported by `pip`](https://pip.pypa.io/en/stable/reference/pip_install/#git).